host/ndc/ndc_integration_test.go

```diff
@@ -373,6 +373,68 @@ func (s *nDCIntegrationTestSuite) TestMultipleBranches() {
 	}
 }
 
+func (s *nDCIntegrationTestSuite) TestEmptyVersionAndNonEmptyVersion() {
```

Comment:
does this [change](https://github.com/temporalio/temporal/pull/2450/files#diff-ed6c10a928ab079f648b25fef4138e0f575a9e0bfe98b242c083d0dcd9e0fafaR212) cover this test?